src/ILVerification/src/TypeVerifier.cs

```diff
@@ -84,7 +86,7 @@ public void VerifyInterfaces()
                 }
                 else
                 {
-                    VerificationError(VerifierError.InterfaceImplHasDuplicate, Format(type), Format(interfaceType));
+                    VerificationError(VerifierError.InterfaceImplHasDuplicate, Format(type), Format(imo.DefType, imo.InterfaceImplementationHandle));
```

Comment:
Actually, I do not think that passing InterfaceImplementationHandle here is correct. It may be a token for a different module.

Reply:
I'll work on it and let you know!Thank's for your time.

Reply:
@jkotas I don't know if I understood well your thought...I change code to print token from TypeDef/Ref, here we can see valid TypeRef
![image](https://user-images.githubusercontent.com/7894084/50846305-19ffc180-136f-11e9-93c7-2af69847913c.png)
![capture2](https://user-images.githubusercontent.com/7894084/50846310-1cfab200-136f-11e9-8289-1ef876811a62.PNG)

Or do you mean if TypeRef load other module and print table metadata token in that module?